test/jest-unit.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   },
   testRegex: ".*\\.spec\\.ts$",
   testSequencer: "<rootDir>/test/jest-test-sequencer.js",
+  setupFilesAfterEnv: ["<rootDir>/test/jest.setup.js"],
```

Comment:
We should not have this in unit testing, if this is needed then is something wrong with code/dependencies

Reply:
Fixed